internal/util/types.go

```diff
@@ -0,0 +1,8 @@
+package util
+
+type contextKey string
+
+// Context keys.
+const (
```

Comment:
This is used in only one place as of now, but it'll be useful when adding the health package and logs in metrics/stackdriver